.github/CODEOWNERS

```
@@ -669,6 +669,7 @@ peps/pep-0788.rst  @ZeroIntensity @vstinner
 # ...
 peps/pep-0789.rst  @njsmith
 peps/pep-0790.rst  @hugovk
+peps/pep-0791.rst  @skirpichev @vstinner
```

```suggestion
peps/pep-0791.rst  @vstinner
```
Per PEP 1: Update [.github/CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) such that any co-author(s) or sponsors with write access to the [PEP repository](https://github.com/python/peps) are listed for your new file. This ensures any future pull requests changing the file will be assigned to them.

@vstinner probably, I shouldn't be here, right?